src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -46,6 +46,7 @@
                 <button
                     type="button"
                     class="btn btn-white"
+                    :disabled="downloadingFiles"
```

Comment:
There isn't an explanation for this change, so asking here: is this cosmetic, is it required to solve the erroneous upload file count? Either way, if we're making this change then does it make sense to do the same for the "Ok" button? Otherwise I can keep clicking it during upload - not sure if this could cause an actual problem, but is inconsistent, and slightly confusing.

Reply:
Now that the change has gone in to recreate the payload on each confirmUpload, it should be safe to allow the user to Cancel at any time, shouldn't it? 

I agree that it would make more sense to disable the OK button while Download is happening, if only as further confirmation to user that something is happening.

Reply:
Cancelling does not really cancel the download or upload, it only closes the modal. If we truly want a user to cancel then we should have cancel button abort download / upload. 

The above is the reason why cancel button should also be disabled while downloading...

Reply:
I see, yes, it's just cancelling the modal, so misleading to still have it enabled once the upload has been (irrevocably) started.